flatsurf/geometry/categories/translation_surfaces.py

```diff
@@ -534,9 +530,23 @@ def erase_marked_points(self):
             function::
 
                 sage: O = flatsurf.translation_surfaces.regular_octagon()
-                sage: O.erase_marked_points() is O
+                sage: O.erase_marked_points().codomain() is O
                 True
 
+            This method produces a morphism from the surface with marked points
+            to the surface without marked points::
+
+                sage: G = SymmetricGroup(4)
+                sage: S = flatsurf.translation_surfaces.origami(G('(1,2,3,4)'), G('(1,4,2,3)'))
+                sage: erasure = S.erase_marked_points()  # optional: pyflatsurf  # long time (1s)
+                sage: marked_point = S(1, 1); marked_point  # optional: pyflatsurf  # long time (from above)
+                Vertex 0 of polygon 2
+                sage: erasure(marked_point)  # optional: pyflatsurf  # long time (from above)
+                sage: unmarked_point = S(1, 0); unmarked_point  # optional: pyflatsurf  # long time (from above)
+                Vertex 0 of polygon 1
+                sage: erasure(unmarked_point)  # optional: pyflatsurf  # long time (from above)
+                Point (-1, 0) of polygon (-3, -7, -2)
```

Comment:
Probably not terribly impressive @videlec but we can now map points in some generality. Here this is

`from_pyflatsurf ⋅ delaunay_triangulation ⋅ erase_marked_point ⋅ delaunay_triangulation ⋅ to_pyflatsurf ⋅ triangulation`